lib/Client.js

```diff
@@ -1288,7 +1294,9 @@ var Client = (function () {
 				for (var _iterator9 = data.presences[Symbol.iterator](), _step9; !(_iteratorNormalCompletion9 = (_step9 = _iterator9.next()).done); _iteratorNormalCompletion9 = true) {
 					var presence = _step9.value;
 
-					self.getUser("id", presence.user.id).status = presence.status;
+					var user = self.getUser("id", presence.user.id);
+					user.status = presence.status;
+					user.gameId = presence.game_id;
```

Comment:
This will load the initial state of gameId when "ready" event is fired, otherwise there's no way to catch the gameId.